content/en/apps/guides/data/analytics/building-dbt-models.md

```diff
@@ -0,0 +1,233 @@
+If they can be public, a branch of the public repository can be created.
+
+The url is set in the `CHT_PIPELINE_BRANCH_URL` environment variable, either in .env if using docker compose, or in values.yaml if useing kubernetes.
+
+When models are changed, CHT Sync needs to be restarted for the change to take effect; when it restarts, it automatically downloads the latest version of cht pipeline and applies than changes; for models materialized as views, they are are updated instantly, for models materialized as incmrenetal tables, the table is dropped and will be unavailable until it is rebuilt from scratch.
+
+## Base Models
+
+The base models define the data which is common to all CHT applications.
+All tables contain a uuid which is the primary key for the table; it is also the `_id` from the source couchdb document.
+
+{{< figure src="cht-pipeline-er.png" link="cht-pipeline-er.png" class=" center col-16 col-lg-12" >}}
+
+### `couchdb`
+All documents are stored in the couchdb db table; downstream models move fields from the document into fields and index them.
+It is possible to get any field from a couchdb document using this table; it is recommended for performance reasons to instead use the columns from downstream models.
+|Field|Description|
+|--|--|
+|`uuid`|CouchDB's unique identifier of the record|
+|`type`|The general type of the document, see below|
+|`doc`| JSON of the source document|
+
+### `data_record`
+All form responses are stored in the `data_record` table, see more details [here]({{< ref "core/overview/db-schema#reports" >}}).
+This table contains columns for the contact who made the report, the parent of that contact, the report data, and a copy of fields in jsonb format.
+It is not recommended to use the fields column directly, but instead to add one new model for each form, moving the relevant fields for that form into columns
+
+|Field|Description|
+|--|--|
+|`uuid`|Unique identifier of the record. Note this is both the primary key for this table, and a foreign key to the couchdb table.|
+|`contact_uuid`| uuid of the `contact` who made the report|
+|`parent_contact_uuid`| uuid of the parent of `contact` who made the report (usually a place)|
+|`fields`| JSON of the `fields` property from the couchdb document|
+|`reported`| the reported timestamp from the couchdb document, stored as a date|
+
+For SMS forms, there is also an `sms_form` table which contains the raw message and sender phone number (for sms form response, `contact_uuid` may be NULL)
+The `contact_uuid` column contains a foreign key to the contacts table.
+
+### `contacts`
+See a description of contact documents in couchdb [here]({{< ref "core/overview/db-schema#contacts-persons-and-places" >}}).
+Every person and place is stored in the contacts table. Persons and patients are stored in their own tables, but because contact types are configurable, other contact types do not have their own tables by default.
```

Comment:
```suggestion
Every person and place is stored in the `contacts` table. Persons and patients are stored in their own tables, but because contact types are configurable, other contact types do not have their own tables by default.
```